Bindings/Java/Matlab/Hopper_Device/BuildDevice.m

```diff
@@ -123,21 +124,15 @@
             for i = 1:size(control,2)
                 controlFunction.addPoint(control(1,i), control(2,i));
             end
-            controller.addActuator(pathActuator);
-            controller.prescribeControlForActuator('cableAtoB',controlFunction);
+            controller.prescribeControlForActuator(name,controlFunction);
         end
         device.addComponent(controller)
         
     case 'passive'
-        name = 'cableAtoB';
+        name = 'cableAtoBpassive';
         springDissipation = 0.1; 
         relaxationTau = 5;
-        
-        if passivePatellaWrap
-            spring0 = 0.7166;
-        else
-            spring0 = 0.6329;
-        end
+        spring0 = 0;
```

Comment:
Actually, this variable represents the `initialStretch` of the spring, so a setting of 0 makes it unstretched in both cases initially (and before it was stretched). Could rename to `stretch0` or `initialStretch` to make it clearer.